src/Themes/VuetifyTheme/Views/Widget-VDivider.liquid

```diff
@@ -0,0 +1 @@
+<v-divider></v-divider>
```

Comment:
We can add props to v-divider for additional customization: https://dev.vuetifyjs.com/en/api/v-divider/#props